package.json

```diff
@@ -12,9 +13,11 @@
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
+    "react-google-button": "^0.7.2",
```

Comment:
A package just to render a button? Please, no. Remove this package. 
You can create a similar button using CSS. Use flexbox.

Reply:
Actually, I thought it would provide some extra functionalities for signing in with Google. 
If you allow can I use this package? It would  make it easy for us to authenticate the user with google sign in
 https://www.npmjs.com/package/react-google-login.
Otherwise I will make one using css . :)

Reply:
For now, I would allow you to use `react-google-login`.
Please remove `react-google-button` package. Code the button manually if needed.

Reply:
okayy..sure.Thank you .
Sorry,That eyes emoji was wrongly sent.

> For now, I would allow you to use `react-google-login`.
> Please remove `react-google-button` package. Code the button manually if needed.